.ci/variables/README.md

```diff
@@ -2,7 +2,7 @@
 
 ```
 export CUSTOMER=cycloid-owl
-for env in "latest" "develop" "a2.5" "a2.6" "a2.7" "py3-a2.6" "py3-a2.7" "py3-a2.8"; do
+for env in "latest" "develop" "py3-a2.6" "py3-a2.8"; do
```

Comment:
readme still used ? If yes py3-a2.7 and 2.9 are missing

Reply:
Ok just checked, since we are using config repo it shouldn't be needed and we might move .ci/variables into our CR (https://github.com/cycloidio/cycloid-stacks/tree/owl/docker-cycloid-toolkit/pipeline)

Reply:
`latest` is `py3-a2.7` now
There is no existing variable for 2.9.

Reply:
oh sneaky 2.9 one.
We should have the variables at the same place

Reply:
yes agree, since config repository works, variables should be in the CR to allow refresh pipeline

Reply:
variables moved to the config repo